hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/protocolPB/RefreshAuthorizationPolicyProtocolClientSideTranslatorPB.java

```diff
@@ -21,17 +21,17 @@
 import java.io.Closeable;
 import java.io.IOException;
 
-import org.apache.hadoop.ipc.ProtobufHelper;
 import org.apache.hadoop.ipc.ProtocolMetaInterface;
 import org.apache.hadoop.ipc.RPC;
 import org.apache.hadoop.ipc.RpcClientUtil;
 import org.apache.hadoop.security.authorize.RefreshAuthorizationPolicyProtocol;
 import org.apache.hadoop.security.proto.RefreshAuthorizationPolicyProtocolProtos.RefreshServiceAclRequestProto;
-import org.apache.hadoop.security.protocolPB.RefreshAuthorizationPolicyProtocolPB;
 
 import org.apache.hadoop.thirdparty.protobuf.RpcController;
 import org.apache.hadoop.thirdparty.protobuf.ServiceException;
 
+import static org.apache.hadoop.ipc.internal.ShadedProtobufHelper.getRemoteException;
```

Comment:
A minor nit. There is an inconsistency where the other helper methods in `ShadedProtobufHelper` are invoked without static import, but for some reason `getRemoteException()` is an exception. :-)

Reply:
switched back in this class. generally i was trying to make the diff smaller; must have got a bit over-enthusiastic here.

Reply:
actually, done something more elegant